README.md

```diff
@@ -27,13 +27,13 @@ It's modeled after Python's `json.tool`, reading from stdin and writing to stdou
   * [FAQ](#faq)
   * [Definitions](#definitions)
   * [Directives](#directives)
-     * [@optional](#optional)
-     * [@output](#output)
-     * [@fold](#fold)
-     * [@tag](#tag)
-     * [@filter](#filter)
-     * [@recurse](#recurse)
-     * [@output_source](#output_source)
+     * [@optional](#-optional)
```

Comment:
Why make these changes? They seem to affect documentation unrelated to the info being added.

Reply:
The links were broken. Just `#optional` doesn't go anywhere. The correct link is `#-optional`.

Reply:
I take it back. This only happens on my browser for some reason :/